Resources/Prototypes/_StarLight/Roles/salaries.yml

```
@@ -8,6 +8,8 @@
     ResearchDirector: 130
     ChiefMedicalOfficer: 135
     Quartermaster: 120
+    BlueShield: 180 #Sent straight from NT, has better wages than station-side employees
+    NanoTrasenRepresentative: 200 #Technically the most important "job" on station.
```

```suggestion

    BlueShield: 180
    NanoTrasenRepresentative: 200
```

Don't add unnecessary comments

---

aight

---

```suggestion
    BlueShield: 110 #Sent straight from NT, has better wages than station-side employees
```
Money is paid for responsibility.

Everyone in command has subordinates.

BSO is a bodyguard, and a bodyguard shouldn’t earn more than the people they’re protecting.

---

fair